models/migrations/v54.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"code.gitea.io/gitea/models"
+
+	"github.com/go-xorm/xorm"
+)
+
+func addModeToDeploKeys(x *xorm.Engine) error {
+	type DeployKey struct {
+		Mode models.AccessMode `xorm:"NOT NULL DEFAULT 1"`
+	}
+
+	if err := x.Sync2(new(DeployKey)); err != nil {
+		return fmt.Errorf("Sync2: %v", err)
+	}
+
+	_, err := x.Cols("mode").Update(&DeployKey{
```

Comment:
Is this really needed as NOT NULL DEFAULT when adding column should automatically add default value